generators/add-action/customer-profile/index.js

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+const path = require('path')
+const ActionGenerator = require('../../../lib/ActionGenerator')
+
+class CustomerProfileGenerator extends ActionGenerator {
+  constructor (args, opts) {
+    super(args, opts)
+    this.props = {
+      description: 'This is a sample action showcasing how to access an external Customer Profile API',
+      // eslint-disable-next-line quotes
+      requiredParams: `['tenantId', 'iMSOrgId', 'apiKey']`,
```

Comment:
could you rename iMSOrgId to orgId and tenantId to tenant (everywhere)?

Reply:
Changed